docs/gettinginvolved/benchmarking.rst

```diff
@@ -115,6 +115,14 @@ Scenario Types
 Operation Types
 ~~~~~~~~~~~~~~~
 
+.. option:: create-dataset
+
+   Create a dataset and wait for cluster to converge.
```

Comment:
See comment above about replacing "wait for cluster to converge", probably applies here too.